src/Twemoji/index.js

```diff
@@ -25,6 +26,7 @@ export default class Twemoji extends React.Component {
 
   render() {
     const { children, ...other } = this.props;
+    delete other.options;
     return <div {...other}>{children}</div>;
```

Comment:
@ZxMYS it will throw a warning. See: https://facebook.github.io/react/warnings/unknown-prop.html